config/locales/api/en.yml

```diff
@@ -445,14 +445,37 @@ en:
         cohort_title: "Result from last visit in %{period}"
         cohort_subtitle: "%{number_of_patients} %{patient_or_patients} registered in %{registration_period}"
         cohort_bar_tooltip: "%{total_patients} %{patient_or_patients} with %{threshold}"
-  monthly_screening_reports:
+  questionnaire_layout:
     male: "Male"
```

Comment:
While designing a questionnaire, it will be simpler to refer to all keys by one kind of naming scheme. This introduces two places where someone might have to lookup a key. 

It should be okay to repeat the strings across these questionnaires if it doesn't cost us much. wdyt?

Reply:
Strings will get repeated eventually as there is not automated check to flag duplications. The aim of this common key isn't to discourage or disallow duplications.

I tried to reuse as many as possible by virtue of a common key just to keep the size of `en.yml` from exploding.